packages/medusa/src/api/routes/store/payment-collections/authorize-payment-collection.ts

```diff
@@ -0,0 +1,15 @@
+import { PaymentCollectionService } from "../../../../services"
+
+export default async (req, res) => {
+  const { payment_id } = req.params
+
+  const paymentCollectionService: PaymentCollectionService = req.scope.resolve(
+    "paymentCollectionService"
+  )
+
+  const payment_collection = await paymentCollectionService.authorize(
```

Comment:
nit(non-blocking): Camel case to follow general convention
```suggestion
  const paymentCollection = await paymentCollectionService.authorize(
```